recipes/pyparallel/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pyparallel" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b5550293af42a42d7b2e1ada1224d3c3ce2f09b80e85421820e068655908c611
+
+build:
+  number: 0
+  # Note: --no-deps is currently required due to https://github.com/conda/conda-build/issues/3254
+  # Once resolved, it should be removed.
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - parallel
+
+about:
+  home: https://github.com/pyserial/pyparallel
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
Please raise an issue upstream to request them to package the license file.

Reply:
The license is now included in `pyparallel:master`.